types/shared/Ownership.ts

```diff
@@ -47,21 +48,24 @@ export type Owners = OwnersNoticeGiven | OwnersNoNoticeGiven | OwnersNoticeDate;
 
 export interface BaseOwners {
   name: string;
-  address: Address | string;
-  interest?: OwnersInterest;
+  address: Address;
```

Comment:
Tidying up this `Address | string` is a direct side effect of the List component now supporting "Address" fields (eg no long text for multiple owners paths!) :tada: